src/main/java/ro/redteam/taskmanagementsystem/services/UserServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package ro.redteam.taskmanagementsystem.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ro.redteam.taskmanagementsystem.exceptions.EmailExistsException;
+import ro.redteam.taskmanagementsystem.exceptions.EmptyInputException;
+import ro.redteam.taskmanagementsystem.models.dtos.UserDTO;
+import ro.redteam.taskmanagementsystem.models.entities.User;
+import ro.redteam.taskmanagementsystem.repositories.UserRepository;
+
+@Service
+@Slf4j
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+    private final ObjectMapper objectMapper;
+
+    public UserServiceImpl(UserRepository userRepository, ObjectMapper objectMapper) {
+        this.userRepository = userRepository;
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public UserDTO createUser(UserDTO userDTO) {
+        if (userDTO.getEmail().isEmpty() || userDTO.getFirstName().isEmpty() || userDTO.getLastName().isEmpty()) {
+            throw new EmptyInputException("Filed empty");
+        }
+
+        if (userRepository.existsByEmail(userDTO.getEmail())) {
```

Comment:
should be used use the insert in data base, without the verification of the unique email, and if an exception occur should be treated according